po/ko.po

```diff
@@ -17336,8 +17571,9 @@ msgid "/// Parity error.\n"
 msgstr ""
 
 #: src/exercises/bare-metal/rtc.md:276
+#, fuzzy
 msgid "/// Break error.\n"
-msgstr ""
+msgstr "/// 새 Counter를 만듭니다.\n"
```

Comment:
again, wrong. this is a translation for "creating a new counter".

Reply:
Fuzzy match, this will be addressed by the Bare Metal PR.